ajax/libs/phaser-ce/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "phaser-ce",
+  "version": "2.7.10",
+  "description": "Phaser CE (Community Edition) is a fast, free and fun HTML5 Game Framework for Desktop and Mobile web browsers.",
+  "filename": "phaser.min.js",
+  "author": {
+    "name": "Richard Davey",
+    "email": "rdavey@gmail.com",
+    "url": "http://www.photonstorm.com"
+  },
+  "homepage": "https://phaser.io",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://photonstorm@github.com/photonstorm/phaser-ce.git"
```

Comment:
Can this url work?
How do you test it?
Please tell me so I can test it next time. Thanks.